service/http/request.go

```diff
@@ -170,6 +170,9 @@ func (r *Request) contentType() int {
 
 // uri fetches full uri from request in a form of string (including https scheme if TLS connection is enabled).
 func uri(r *http.Request) string {
+	if r.URL.Host != "" {
```

Comment:
So, in this case, we are fully relying on URL provided by parent service?

Reply:
Yes, I've investigated it. FastCGI returns full URL